This patch is for Python 3 compatibility.  It has not been submitted upstream,
as the community appears to have gone dormant.

--- filechunkio-1.5/setup.py~	2011-04-27 06:12:34.000000000 -0700
+++ filechunkio-1.5/setup.py	2014-04-16 11:32:41.158068141 -0700
@@ -1,12 +1,11 @@
 #!/usr/bin/env python
 from distutils.core import setup
 
-from filechunkio import __version__
 
 
 setup(
     name="filechunkio",
-    version=unicode(__version__),
+    version="1.5",
     description="FileChunkIO represents a chunk of an OS-level file "\
         "containing bytes data",
     long_description=open("README", 'r').read(),
